proknow-sdk/Requestor.cs

```diff
@@ -308,7 +311,15 @@ internal async Task<string> GetDomainStatusAsync()
                     {
                         queryParameters = new Dictionary<string, object>();
                     }
-                    queryParameters["next"] = next;
+                    if (isPatientsRoute)
+                    {
+                        queryParameters["page_newest"] = DateTime.UtcNow.ToString("o");
```

Comment:
Might want to follow my comment on the Python PR: https://github.com/proknow/proknow-python/pull/77#discussion_r1088923582